config/webpack.config.js

```diff
@@ -1,9 +1,9 @@
 // Staging config. Also the default config that prod and dev are based off of.
-const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
```

Comment:
We'd been holding off on upgrading to this plugin because it didn't appear necessary, but it improves performance with v4.